examples/with-jest-and-jsdom/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
What is it intended to extend? Looks like it references a file that lives outside of the example. Let's keep all the examples self-contained.

Reply:
This is the same code as what is in examples/with-jest.